File: official/docs/java/shipments/RetrieveSmartrate.java
import com.easypost.EasyPost;

public class RetrieveSmartrate {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Shipment shipment = Shipment.retrieve("shp_...");

        shipment.getSmartrates();

        System.out.println(shipment);
    }
}

File: official/docs/node/shipments/retrieve_smartrate.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

api.Shipment.retrieve('shp_...').then((retrieveShipment) => {
  retrieveShipment.getSmartrates().then(console.log);
});
